src/main/java/com/stripe/net/RawApiRequest.java

```diff
@@ -1,23 +1,51 @@
 package com.stripe.net;
 
+import java.util.ArrayList;
+import java.util.List;
 import lombok.Getter;
 
 public class RawApiRequest extends BaseApiRequest {
-
   @Getter(onMethod_ = {@Override})
   private RawRequestOptions options;
 
   @Getter private String rawContent;
 
+  private RawApiRequest(
+      BaseAddress baseAddress,
+      ApiResource.RequestMethod method,
+      String path,
+      RawRequestOptions options,
+      ApiMode apiMode,
+      List<String> usage,
+      String rawContent) {
+    super(baseAddress, method, path, options, apiMode, usage);
+    this.rawContent = rawContent;
+    this.options = options;
+  }
+
   public RawApiRequest(
       BaseAddress baseAddress,
       ApiResource.RequestMethod method,
       String path,
       String rawContent,
       RawRequestOptions options,
       ApiMode apiMode) {
-    super(baseAddress, method, path, options, apiMode, null);
-    this.rawContent = rawContent;
-    this.options = options;
+    this(baseAddress, method, path, options, apiMode, null, rawContent);
+  }
+
+  public RawApiRequest addUsage(String usage) {
+    List<String> newUsage = new ArrayList<>();
+    if (this.getUsage() != null) {
+      newUsage.addAll(this.getUsage());
+    }
```

Comment:
nit: Maybe add a test to check that existing usage is retained?

Reply:
Done in [Add test for .addUsage](https://github.com/stripe/stripe-java/pull/1731/commits/ccdb9f10a2c69dc7a00898968f1a12672aba6f44)